src/data/real/sqrt.lean

```diff
@@ -307,6 +308,24 @@ theorem neg_sqrt_lt_of_sq_lt (h : x^2 < y) : -sqrt y < x := (sq_lt.mp h).1
 
 theorem lt_sqrt_of_sq_lt (h : x^2 < y) : x < sqrt y := (sq_lt.mp h).2
 
+/- The natural square root is at most the real square root -/
+lemma nat_sqrt_le_real_sqrt {a : ℕ} : ↑(nat.sqrt a) ≤ real.sqrt ↑a :=
+begin
+  rw real.le_sqrt,
+  norm_cast,
+  exact nat.sqrt_le' a,
+  simp, simp,
```

Comment:
Please avoid non-terminal simps: https://leanprover-community.github.io/glossary.html#non-terminal-simp

Reply:
Makes sense! Is there a way to easily dump what `simp` did?

Reply:
`squeeze_simp`

Reply:
Also, please wrap subproofs (i.e., when there's > 1 goal) in `{ .. }`.